lib/database_cleaner/active_record/truncation.rb

```diff
@@ -1,18 +1,20 @@
 require "delegate"
+require 'database_cleaner/active_record/base'
 
 module DatabaseCleaner
   module ActiveRecord
     class Truncation < Base
       def initialize(opts={})
-        if !opts.empty? && !(opts.keys - [:only, :except, :pre_count, :cache_tables, :reset_ids]).empty?
-          raise ArgumentError, "The only valid options are :only, :except, :pre_count, :reset_ids, and :cache_tables. You specified #{opts.keys.join(',')}."
+        if !opts.empty? && !(opts.keys - [:only, :except, :pre_count, :cache_tables, :reset_ids, :truncate_option]).empty?
+          raise ArgumentError, "The only valid options are :only, :except, :pre_count, :reset_ids, :cache_tables and :truncate_option. You specified #{opts.keys.join(',')}."
         end
 
         @only = Array(opts[:only]).dup
         @except = Array(opts[:except]).dup
 
         @reset_ids = opts[:reset_ids]
         @pre_count = opts[:pre_count]
+        @truncate_option = opts[:cascade] || :restrict
```

Comment:
I'm pretty sure this should be `opts[:truncate_option]` rather than `opts[:cascade]` (I know this is from the original PR - just want to make sure it gets addressed before merging!)